### PR TITLE
Title field on item model

### PR DESCRIPTION
**Pull request by warrd**
## Issue

https://github.com/wellcometrust/platform/issues/4101

## Description

As part of the METS work we are bringing across more data for individual items that make up a work. Here we include a new `title` field on `Item`, which is populated from the Sierra data. This will be needed to properly display the individual items on the frontend (similarly to how items are displayed on the [library site](http://search.wellcomelibrary.org/iii/encore/record/C__Rb2975324?lang=eng))

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@warrd where are we getting this from, and where is it being used?

**Comment by warrd** [comment]
@jamesgorrie still working on this, the sierra transformer will populate it. Will add a description when to the PR when ready.

**Comment by jamesgorrie** [comment]
Thanks @warrd, sorry to jump the 🔫

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraItems.scala [hunk 1]
```diff
@@ -114,4 +93,31 @@ case class SierraItems(itemDataMap: Map[SierraItemNumber, SierraItemData])
       None
     }
   }
+
+  private def transformItemData(
+    itemId: SierraItemNumber,
+    itemData: SierraItemData): Identifiable[Item] = {
+    debug(s"Attempting to transform $itemId")
```
Comment (jamesgorrie) [inline pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraItems.scala]: Want to keep this?

**Comment by alicefuzier** [comment]
@jamesgorrie It's the "Envelope", "0", "1", etc after the shelfmark.
As we start bringing in more item information with METS, this helps to distinguish which item is which from a user point of view

File: pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraItems.scala (hunk 1 above)
Comment (warrd) [reply pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraItems.scala]: This was from the original code (which was copy / pasted), I'm not that bothered about it myself but could be useful

**Comment by jamesgorrie** [comment]
@alicefuzier completely missed that - thanks!

Do you know where the holdings information comes from?
> Complete set of 16 prints and original envelope

holdings

**Comment by kenoir** [comment]
This'll be handy in the requesting service too 👍

**Pull request merged**